pkg/kube/inject/testdata/inject/app_probe/hello-probes-with-flag-unset-in-annotation.yaml.injected

```diff
@@ -183,6 +183,8 @@ spec:
         volumeMounts:
         - mountPath: /var/run/secrets/istio
           name: istiod-ca-cert
+        - mountPath: /var/lib/istio/data
```

Comment:
seeing a pattern here. why have the var/lib/istio/data dir? Is it because data is an emptydir and you don't wish to overwrite /var/lib/istio?

Reply:
Yes, that is correct. We want data to be an empty dir so that it can survive a container restart.